### PR TITLE
Build system tweaks from downstream

### PR DESCRIPTION
**Pull request by thesamesam**
Hi,

These are a few patches we've been maintaining downstream in Gentoo which should help general distribution folks out.

### CODE REVIEW

File: libopeniscsiusr/Makefile [hunk 1]
```diff
@@ -108,7 +108,7 @@ docs/man/$(EXTRA_MAN_FILES).gz: $(HEADERS)
 	    perl docs/doc-preclean.pl > "$(TEMPFILE)"
 	perl docs/kernel-doc -man "$(TEMPFILE)" | \
 	    perl docs/split-man.pl docs/man
-	-rm -f "$(TEMPFILE)"
+	-$(RM) -f "$(TEMPFILE)"
```
Comment (gonzoleeman) [inline libopeniscsiusr/Makefile:111]: The RM Make macro is already set to "rm -f" by default, so you need not keep the "-f". Please fix this here and in other Makefiles.

**Review by gonzoleeman — CHANGES_REQUESTED** [review]
Except for the unneeded "-f" in "$(RM) -f", looks good. I'd be glad to merge these with that change.

**Comment by thesamesam** [comment]
Yep, I'll be doing it later. Yesterday I was busy mostly IRL.

File: libopeniscsiusr/Makefile (hunk 1 above)
Comment (thesamesam) [reply libopeniscsiusr/Makefile]: Done, thanks!

**Comment by thesamesam** [comment]
Done.

**Review by gonzoleeman — APPROVED** [review]

**Comment by gonzoleeman** [comment]
Thank you!

**Comment by thesamesam** [comment]
Thanks!

**Pull request merged**